### PR TITLE
UI/jose/UI converter

### PR DESCRIPTION
**Pull request by Git-Tonio**

### CODE REVIEW

File: app/src/main/java/ju/video/player/App.java [hunk 1]
```diff
@@ -3,29 +3,18 @@
...
-import ju.video.player.view.MetaDataFrame;
 import ju.video.player.view.playlist.PlayListFrame;
-import ju.video.player.view.videoconvert.VideoConvertFrame;
-import ju.video.player.view.videoplayer.*;
 
 import java.io.IOException;
 
 public class App {
     public static void main(String[] args) throws IOException, ParseException {
-        //Convert convert = new Convert();
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/App.java:18]: Please, delete the comments

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 2]
```diff
@@ -1,86 +1,81 @@
...
 import ju.video.player.view.materialDesing.components.utils.RoundedBorder;
 import ju.video.player.view.materialDesing.display.FrameUtility;
-import ju.video.player.view.materialDesing.layouts.ResponsiveLayout;
-import java.awt.event.ActionListener;
-import java.awt.Dimension;
 
 import javax.swing.JButton;
-
 import javax.swing.*;
 import java.awt.*;
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java:14]: Please specify the Javax Swing and Java awt libraries used.

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 3]
```diff
@@ -1,86 +1,81 @@
...
     static JFrame f;
 
     // label
     static JLabel l, l1, l3, l4;
+    //
 
     
     // combobox
-    static JComboBox c1, c2;
+    private JComboBox c1;
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: Please be more specific in the variable names.

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 4]
```diff
@@ -1,86 +1,81 @@
...
-
-import javax.swing.JButton;
-
 public class VideoConvertFrame extends JFrame{
     // frame
     static JFrame f;
 
     // label
     static JLabel l, l1, l3, l4;
+    //
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java:22]: Please eliminate unnecessary spaces

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 5]
```diff
@@ -1,86 +1,81 @@
...
+        this.path = path;
         initFrame();
     }
 
     public void initFrame() {
         JFrame frame = FrameUtility.build("ATT Player", 0, 0, 850, 600, true);
-
+        
+        JButton btnV= buttonConv();
+        // btnV.addActionListener(ActionListener listener);
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: Please delete unnecessary comments

File: app/src/main/java/ju/video/player/view/playlist/playlistpanel/MediaPanel.java [hunk 6]
```diff
@@ -16,24 +16,29 @@
...
 import ju.video.player.controller.PlayMediaButtonController;
 import ju.video.player.view.commons.Button;
 import ju.video.player.view.commons.Colors;
 
-public class MediaPanel extends JPanel{
+public class MediaPanel extends JPanel {
 
     public MediaPanel(String mediaName, int index) {
         initPanel();
+        // System.out.println(mediaName);
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/playlist/playlistpanel/MediaPanel.java:28]: Use Logger instead the System.out.print

**Review by junior0123 — CHANGES_REQUESTED** [review]

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 7]
```diff
@@ -1,86 +1,81 @@
...
-import java.awt.Dimension;
-
-import javax.swing.JButton;
-
 public class VideoConvertFrame extends JFrame{
     // frame
     static JFrame f;
 
     // label
     static JLabel l, l1, l3, l4;
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: please, select a properly name for the variables

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 8]
```diff
@@ -1,86 +1,81 @@
...
+
+
+    public VideoConvertFrame(String path) {
         controller = new ConvertController(this);
+        this.path = path;
         initFrame();
     }
 
     public void initFrame() {
         JFrame frame = FrameUtility.build("ATT Player", 0, 0, 850, 600, true);
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: avoid the magic numbers

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 9]
```diff
@@ -1,86 +1,81 @@
...
 
     }
 
-    public JButton buttonConv(ActionListener listener) {
+    public JButton buttonConv() {
         JButton Button = new JButton();
         RoundedBorder border = new RoundedBorder(20);
 
         Button.setBorder(border);
         Button.setBounds(300, 300, 260, 50);
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: Avoid the magic numbers

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 10]
```diff
@@ -1,86 +1,81 @@
...
+       
         Button.setVisible(true);
         return Button;
 
     }
 
     public JPanel panelSelect() {
 
-        String s1[] = { "AVI", "MKV", "FLV" };
+        String s1[] = { "avi", "mkv", "flv" };;
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: Choose an other name for the variable

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 2 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java:14]: The libraries have already been changed to the specific ones.

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 3 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: the variable names have already been changed to more meaningful ones.

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 4 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java:22]: the variable names have already been changed to more meaningful ones.

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 5 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: I deleted the unnecessary comments

**Review by junior0123 — APPROVED** [review]

**Review by AdriOli134 — CHANGES_REQUESTED** [review]
Please, don't forget to add copyright and the documentation of the methods

**Pull request closed without merging**